pkgs/applications/virtualization/singularity/generic.nix

```diff
@@ -0,0 +1,198 @@
+# Configurations that should only be overrided by
+# overrideAttrs
```

Comment:
I got out of sync and need to think again what the current code does, but an immediate note about this comment:  `overrideAttrs` overrides arguments to `mkDerivation` (i.e. what goes into `{ ... }` in `mkDerivation { ... }`) and not `callPackage` arguments